tests/integration/orders-update.spec.js

```diff
@@ -40,6 +43,7 @@ test('the module should modify an existing order', (t) => {
       ),
     )
     // Create needed data
+    .then(() => ordersUpdate.client.channels.ensure(channelKey, channelRole))
```

Comment:
yeah, just that it was only implement for channels, but yeah we don't need to delete (I wasn't thinking of deleting, not when am not drunk 👻⿎)